Cargo.lock

```diff
@@ -4,8 +4,12 @@ version = "0.1.0"
 dependencies = [
  "bson 0.1.2 (git+https://github.com/kyeah/bson-rs)",
  "byteorder 0.3.10 (registry+https://github.com/rust-lang/crates.io-index)",
+ "libc 0.1.8 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
The guide doesn't seem to say anything explicitly, but they should be listed lexicographically; looks like that's the case in Cargo.toml and lib.rs. Are you talking about module-level imports (`use ...`)?